hieradata/node/mongo-1.backend.staging.publishing.service.gov.uk.yaml

```diff
@@ -1,15 +1,4 @@
 govuk_env_sync::tasks:
-  "push_licence_finder_production":
-    ensure: "present"
```

Comment:
Do you need to change this to `ensure: "absent"` and deploy gouvk-puppet before removing this Hieradata?

Reply:
yeah you are right. I will adopt a 2-step approach: 1. absent and 2. remove hiera later.